torchao/quantization/quantize_/workflows/_utils.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD 3-Clause license found in the
+# LICENSE file in the root directory of this source tree.
+
+import torch
+
+from torchao.quantization.quantize_.common import QuantizeTensorKwargs
+
+
+def _choose_quant_func_and_quantize_tensor(
```

Comment:
nit: colocate with `QuantizeTensorKwargs` and move out from utils, since this is more of a developer facing function which is central to the design?

Reply:
hmmm, I have thought about move this to `common/` but then common is going to import from the workflow specific code, this seems weird, like:
```
from torchao.quantization.quantize_.workflows import (
        Float8Tensor,
        QuantizeTensorToFloat8Kwargs,
    )
```

any ideas to resolve that?